src/lists.rs

```diff
@@ -317,6 +357,16 @@ pub fn parse_filters(
     (network_filters, cosmetic_filters)
 }
 
+/// Parse an entire list of filters, ignoring any errors
+pub fn parse_filters(
+    list: &[String],
+    debug: bool,
+    format: FilterFormat,
+) -> (Vec<NetworkFilter>, Vec<CosmeticFilter>) {
+
```

Comment:
nit: extra line

@ShivanKaul how'd you manage to insert so many of these? :sweat_smile:

Reply:
Haha sorry - I'm going to blame my editor. We should use a standard format config that we check in.